.yamato/project-standards.yml

```diff
@@ -0,0 +1,20 @@
+{% metadata_file .yamato/project.metafile %}
+---
+standards_{{ projects.first.name }}:
+  name: Standards Check {{ projects.first.name }}
+  agent:
+    type: Unity::VM
+    image: desktop/logging-testing-linux:stable
+    flavor: b1.small
+  commands:
+    - dotnet --version
+    - $HOME/.dotnet/tools/dotnet-format --version
+    - python3 --version
+    - pip install unity-downloader-cli --upgrade --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - unity-downloader-cli -u {{ test_editors.first }} -c editor --wait --fast
+    - .Editor/Unity -batchmode -nographics -logFile - -executeMethod TestProject.Editor.CLI.ThingsForFun -projectPath {{ projects.first.path }} -quit
+    - ls -al
+    - ls -al {{ projects.first.path }}
+    - ls -al .Editor
+    - $HOME/.dotnet/tools/dotnet-format ./{{ projects.first.path }}/{{ projects.first.path }}.sln --fix-whitespace --fix-style error --check
```

Comment:
Wonder about this path? 
> {{ projects.first.path }}/{{ projects.first.path }}.sln
Should the second "projects.first.path" be something like "projects.first.name"? or is there something else behind the scene extracting the name in that last part of the path to the solution?

Reply:
`{{ projects.first.path }}` is `testproject`

Reply:
Right... ok... for some reason my head was assuming the path was full and not relative....
testproject/testproject.sln 
Makes sense!